app/Bus/Commands/Incident/ReportIncidentCommand.php

```diff
@@ -110,7 +110,7 @@
         'component_status' => 'nullable|required_with:component_id|int|min:0|max:4',
         'notify'           => 'nullable|bool',
         'stickied'         => 'required|bool',
-        'incident_date'    => 'nullable|string',
+        'occurredAt'       => 'nullable|string',
```

Comment:
I reckon we should move to `camelCase` but I also reckon it should be done in it's own PR rather than just leaving random bits here and there.